pipeline/metadata/design_table.csv

```diff
@@ -2698,6 +2698,8 @@ DLG-F3_306,Teros21Table,"Teros21(17,2)",DLG,F3,TEROS21,306,6,D4,,soil-temp-30cm,
 DLG-F3_306,Teros21Table,"Teros21(18,1)",DLG,F3,TEROS21,306,6,D4,,soil-vwc-40cm,
 DLG-F3_306,Teros21Table,"Teros21(18,2)",DLG,F3,TEROS21,306,6,D4,,soil-temp-40cm,
 ,,,,,,,,,,,
+Compass_CRC_UP_303,SapflowA,DiffVoltA_Avg({9:12}),DLG,F4,Sapflow,303,"{356,351,343,339}",,,sapflow-3.5cm,Remapped in June 2025 from CRC UP deeps
```

Comment:
Is the new instrument ID here to avoid throwing an error?

Reply:
I think that's right, yes.